modules/pel/peltool/README.md

```diff
@@ -0,0 +1,74 @@
+### peltool usage
+To get more peltool's capabilities, use the help option as shown below.
+`peltool.py --help`
+**Note:** For non-BMC environment user must pass the PEL directory to parse PELs
```

Comment:
I think is not aligned properly. This `Note` should in the new line for better readability.

```
To get more peltool's capabilities, use the help option,`peltool.py --help`.


**Note:** For non-BMC environment user must pass the PEL directory to parse PELs. For example, `peltool.py -p <pel_dir> -l`, `peltool.py -p <pel_dir> -n`, `peltool.py -p <pel_dir> -a``